lumigator/infra/mzai/helm/lumigator/test/custom-mistral-key.yaml

```diff
@@ -0,0 +1 @@
+mistralKey: abcdefgh
```

Comment:
I think you added the files in this folder by mistake, could you check it?

Reply:
Hmm, I'm using this directory to hold test values for the moment. The idea is that the test tool for helm templates will use these values files. But if you feel that these test values files are better placed somewhere else, just let me know.

You can check for example that `helm template -f test/custom-mistral-key.yaml` will generate the secret whereas the one with the Secret name will not.

Reply:
agree with adding to a separate folder (so it doesn't get shipped with the chart

Reply:
Ok, removing these files right now